addons/airway/stringtable.xml

```diff
@@ -319,7 +319,7 @@
             <Norwegian>Luftvei er blokkert, behov for luftveishåndtering</Norwegian>
             <French>Voies aériennes obstruées, geste médical nécessaire</French>
             <Finnish>Hengitystiet ovat tukossa, hengityshoitoa tarvitaan</Finnish>
-            <German>Die Atemwege sind blockiert, eine Atemwegskontrolle ist erforderlich</German>
+            <German>Die Atemwege sind blockiert, Atemwegssicherung ist erforderlich</German>
```

Comment:
Is Atemwegssicherung ein gängeriger Begriff im San Dienst? Atemswegskontrolle scheint mir zumindest viel geläufiger.

Reply:
Die aktion ist bereits die atemwegskontrolle

Reply:
Alternativ wäre dann noch ` ..., die Atemwege müssen freigelegt werden`